tests/meta/upload/vogue/conftest.py

```diff
@@ -36,7 +36,52 @@ def genotype_return_sample():
     return _config
 
 
-@pytest.fixture(scope='function')
+APPTAGS = [
+    {"tag": "RMLP10R825", "category": "wgs"},
+    {"tag": "METLIFR010", "category": "wes"},
+    {"tag": "EXLKTTR080", "category": None},
```

Comment:
category None? Should not be a valid thing I think..

Reply:
vogue will never add a key to its database with a value None. It is handeld over there anyway. But you think its cleaner to remove Nones already in the UploadVogueAPI in cg/meta/upload/vogue?

Reply:
Category is deprecated in favour of prep_category and this value can not be none of I remember correctly

Reply:
Looks like it can. "Tillåt Null"l is checked!


![Skärmavbild 2020-01-08 kl  10 34 02](https://user-images.githubusercontent.com/1306333/71966761-7c99d080-3202-11ea-96a4-cc1eea384b94.png)

Reply:
👍

Reply:
Could we change it to "Tillåt INTE Null" in your PR?